options/m_config.h

```diff
@@ -154,6 +154,7 @@ enum {
     M_SETOPT_NO_FIXED = 64,         // Reject M_OPT_FIXED options
     M_SETOPT_NO_PRE_PARSE = 128,    // Reject M_OPT_PREPARSE options
     M_SETOPT_NO_OVERWRITE = 256,    // Skip options marked with FROM_*
+    M_SETOPT_NO_BLACKLIST = 512,    // Reject options in blacklist
```

Comment:
I would motion to rename this to `M_SETOPT_WATCH_LATER` to clarify when exactly this is set, and to decouple the "this is the watch later stage" signalling from the actual blacklist logic, to keep the blacklist more in one place.

Reply:
Also, "NO BLACKLIST" sounds like it would ignore the blacklist if this flag is set. More reason we should rename it IMO.